plugins/catalog-backend-module-orchestrator-entity-provider/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@janus-idp/backstage-plugin-catalog-backend-module-orchestrator-entity-provider",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module"
+  },
+  "homepage": "https://janus-idp.io/",
+  "repository": "github:janus-idp/backstage-plugins",
+  "bugs": "https://github.com/janus-idp/backstage-plugins/issues",
+  "keywords": [
+    "backstage",
+    "plugin",
+    "orchestrator",
+    "workflows"
+  ],
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "tsc": "tsc",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.19.8",
+    "@backstage/backend-plugin-api": "^0.6.6",
+    "@backstage/config": "^1.1.1",
+    "@backstage/core-plugin-api": "^1.7.0",
+    "@backstage/plugin-catalog-node": "^1.4.7",
+    "@backstage/plugin-events-backend": "^0.2.8",
+    "@backstage/plugin-events-node": "^0.2.8",
+    "@janus-idp/backstage-plugin-orchestrator-common": "0.0.1",
+    "@janus-idp/backstage-plugin-orchestrator-backend": "0.0.1",
+    "@backstage/types": "^1.1.1",
+    "json-schema": "^0.4.0",
```

Comment:
We should avoid this package.
Let's explore the need for it and eventually switch to i.e. https://www.npmjs.com/package/json-schema-library

Reply:
I understand the concern but this lib is also used by Backstage, so at least we are in sync with them. We can re-evaluate replacing this library in a future opportunity.